app/workers/webhook/dispatch_event_worker.rb

```diff
@@ -0,0 +1,12 @@
+module Webhook
+  class DispatchEventWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :medium_priority
+
+    def perform(endpoint_url, event_json)
```

Comment:
I thought I liked the old `payload` better but not I'm unsure hahaha

Reply:
I'm personally a fan of using a hash as arguments bc then it is clear what the params are without having to go into the job in the code to figure it out. But I'm good either way.

Reply:
I though we had abandoned the idead in an old PR (not to use hash arguments) because they are not well supported by Sidekiq: 

> The arguments you pass to perform_async must be composed of simple JSON datatypes: string, integer, float, boolean, null(nil), array and hash. This means you must not use ruby symbols as arguments.

from https://github.com/mperham/sidekiq/wiki/Best-Practices

Reply:
I could go either way here.

I know we've discussed not using hash arguments and [using hash arguments](https://github.com/thepracticaldev/dev.to/pull/5615#discussion_r368736454).

My thinking for doing it this way was based on a few thoughts:
1. Since it's only 2 arguments, it's not quite as confusing as to what's being passed in.
2. When the job is called, you can see what's being passed in since [both arguments are built beforehand just above it](https://github.com/thepracticaldev/dev.to/blob/15ccdcbfba9ad517c0b7dd023cf93072720ed2e1/app/services/webhook/dispatch_event.rb#L12):
```ruby
DispatchEventWorker.perform_async(url, event_json)
```
3. Since `event_json` is a Hash of its own, if we changed it to Hash arguments we'd have a nested Hash that we would probably want to do `deep_symbolize_keys!` on to keep everything consistent in the job. Not a big deal, but felt a little "dirtier" than necessary.

On the other hand, I'm all for the idea that using a Hash argument does improve clarity. I can just about always agree with that point!

Reply:
@rhymes we don't want to attempt to pass symbols, but a hash with string keys when you push past 3 arguments I think is a good idea. Like @atsmith813 mentioned, in this case we only have a couple, it's not a big deal, but the more you get a mapped hash is nice visibility I think. It makes it really easy to see in the dashboard what arguments are what.